### ISSUE TITLE
Cannot export PCK without templates installed/configured

### ISSUE THREAD

**Issue by Ev1lbl0w** [godotengine/godot#64748]
### Godot version

3.5.stable

### System information

Arch Linux

### Issue description

I tried exporting to a lot of platforms and it looks like exporting to PCK is platform-independent:
```bash
$ sha256sum *.pck
9bc8e1cd2ff8dc8155fe107b53c0c4e0a4cf1f693f77e68464a35e557fd0f717  android_armv7.pck
9bc8e1cd2ff8dc8155fe107b53c0c4e0a4cf1f693f77e68464a35e557fd0f717  android_armv8.pck
9bc8e1cd2ff8dc8155fe107b53c0c4e0a4cf1f693f77e68464a35e557fd0f717  html.pck
9bc8e1cd2ff8dc8155fe107b53c0c4e0a4cf1f693f77e68464a35e557fd0f717  ios_arm64.pck
9bc8e1cd2ff8dc8155fe107b53c0c4e0a4cf1f693f77e68464a35e557fd0f717  ios_armv7.pck
9bc8e1cd2ff8dc8155fe107b53c0c4e0a4cf1f693f77e68464a35e557fd0f717  macos.pck
9bc8e1cd2ff8dc8155fe107b53c0c4e0a4cf1f693f77e68464a35e557fd0f717  uwp.pck
9bc8e1cd2ff8dc8155fe107b53c0c4e0a4cf1f693f77e68464a35e557fd0f717  win_32.pck
9bc8e1cd2ff8dc8155fe107b53c0c4e0a4cf1f693f77e68464a35e557fd0f717  win_64.pck
9bc8e1cd2ff8dc8155fe107b53c0c4e0a4cf1f693f77e68464a35e557fd0f717  x11_32.pck
9bc8e1cd2ff8dc8155fe107b53c0c4e0a4cf1f693f77e68464a35e557fd0f717  x11_64.pck
```
However, the button to export to PCK is disabled if no platforms exist, or if a platform has configuration errors: 
![image](https://user-images.githubusercontent.com/6501975/185995021-03ee6167-753e-4423-a5d8-cb5730d4327e.png)

When searching for a similar issue, I found #12040 which should fix this, but I'm not sure if this was never ported to `master` or if this is a regression.

### Steps to reproduce

- Create an empty project
- Try to export it
- The "Export PCK/Zip..." is greyed out until a valid export platform is added and configured.

### Minimal reproduction project

Not needed, only an empty project is required

**Comment by akien-mga** [comment]
This was actually implemented in `master` by @aaronfranke just a couple of weeks ago: #62925.

Could be backported for 3.6 I guess.

**Comment by Ev1lbl0w** [comment]
Awesome! I'd love if this gets backported, as it can get quite annoying when using custom versions of Godot and having to compile one export template just to get rid of it :sweat_smile:

**Comment by akien-mga** [comment]
Fixed by #62925.

**Comment by rambda** [comment]
For any who encountered this problem in 3.5.*, run `godot --export-pack "preset_name" "filename.pck"` from the command line, and you can also add the `--no-window` parameter to run silently in the background.
